internal/operators/cnv/cnv_operator.go

```diff
@@ -0,0 +1,139 @@
+package cnv
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/internal/hardware"
+	"github.com/openshift/assisted-service/internal/operators/api"
+	models "github.com/openshift/assisted-service/models"
+	"github.com/sirupsen/logrus"
+)
+
+type operator struct {
+	log logrus.FieldLogger
+}
+
+// NewCNVOperator creates new instance of a Container Native Virtualization installation plugin
+func NewCNVOperator(log logrus.FieldLogger) *operator {
+	return &operator{
+		log: log,
+	}
+}
+
+// GetType reports type of an operator this Operator manages
+func (o *operator) GetType() models.OperatorType {
+	return models.OperatorTypeCnv
+}
+
+// GetDependencies provides a list of dependencies of the Operator
+func (o *operator) GetDependencies() []models.OperatorType {
+	return []models.OperatorType{models.OperatorTypeLso}
```

Comment:
I think that it should be `models.OperatorTypeOcs`.

Reply:
During the meeting @dankenigsberg suggested we should not relay on OCS but have some kind of storage class. `LSO` always needs to be there for both choices

Reply:
During that meeting I was convinced that any of OCS and LSO should be fine, assuming the latter exposes block storage class. I am not 100% sure of that myself.

Reply:
@fabiand What do you think?